src/sentry/replays/usecases/ingest/issue_creation.py

```diff
@@ -4,17 +4,59 @@
 from uuid import UUID
 
 from sentry.constants import MAX_CULPRIT_LENGTH
-from sentry.issues.grouptype import ReplayRageClickType
+from sentry.issues.grouptype import ReplayHydrationErrorType, ReplayRageClickType
 from sentry.issues.issue_occurrence import IssueEvidence
 from sentry.replays.usecases.issue import new_issue_occurrence
 from sentry.utils import metrics
 
 logger = logging.getLogger()
 
+HYDRATION_ERROR_TITLE = "Hydration Error"
+HYDRATION_ERROR_SUBTITLE = "A hydration error occurred."
+HYDRATION_ERROR_LEVEL = "error"
 RAGE_CLICK_TITLE = "Rage Click"
 RAGE_CLICK_LEVEL = "error"
 
 
+def report_hydration_error_issue_with_replay_event(
+    project_id: int,
+    replay_id: str,
+    timestamp: float,
+    url: str,
+    replay_event: dict[str, Any],
+):
+    metrics.incr("replay.hydration_error_issue_creation_with_replay_event")
+
+    # Seconds since epoch is UTC.
+    date = datetime.datetime.fromtimestamp(timestamp)
+    timestamp_utc = date.replace(tzinfo=datetime.UTC)
+    fingerprint = str(_trunc_timestamp_week(timestamp))
```

Comment:
This will send a notification once per week.  Is that something we want to do?  Should we group all the hydration errors in one issue and let their frequency bump it to the top of the issue queue?

Reply:
Yeah, Ryan was also with you on this, you guys know more about the issues workflow so I'll hard code it to "1"